deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-service-core.ts

```diff
@@ -130,7 +131,13 @@ export class MigrationServiceCore extends Stack {
             portMappings: props.portMappings,
             logging: LogDrivers.awsLogs({
                 streamPrefix: `${props.serviceName}-logs`,
-                logGroup: serviceLogGroup
+                logGroup: serviceLogGroup,
+                // E.g. "[INFO ] 2024-12-31 23:59:59,99999999"
+                // and  "[ERROR] 2024-12-31 23:59:59,999"
+                // and  "2024-12-31 23:59:59"
+                multilinePattern: "^(\\[[A-Z ]{1,5}\\] )?\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2}(,\\d{2,8})?",
```

Comment:
The intention here is to provide a flexible regex for how AWS identifies the start of a new log line based on the starting characters in a line, while trying to decrease the likelihood that some intermediate print statement would match and thus separate from it's associated line. The log4j2.properties defines the actual log pattern.

I'll remove the mili/micro second pattern from this as it's not helping filter any potential patterns since it's an optional match.